cmd/api/protos/tracker-api.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package tracker;
+
+import "google/protobuf/empty.proto";
+
+option go_package = "github.com/doniacld/grpc-in-go/cmd/api/protos";
+
+// MoodType, the possible values for a mood.
+enum MoodState {
+  // Happy feeling
+  Happy = 0;
+  // Sad feeling
+  Sad = 1;
+}
+
+// Mood definition.
+message Mood {
+  // MoodState defines the mood state among defined values
+  MoodState State = 1;
```

Comment:
Should fields have a capital? What is the standard?
I usually use full lowercase to pass Goggle's horribly strong linter.

Reply:
Found the standard [gRPC standard language guide: Adding Comments](https://developers.google.com/protocol-buffers/docs/proto3#adding_comments)